src/Application/GameWindowHandler.cpp

```diff
@@ -29,9 +33,84 @@ using ApplicationIoc = Application::IocContainer;
 using Application::GameWindowHandler;
 using Io::InputAction;
 
+static char PlatformKeyToChar(PlatformKey key, PlatformModifiers mods) {
```

Comment:
doesn't it belong to platform now?

Reply:
Not really. Platform API is intended to be super lean. If something can be done at the next abstraction layer, then this is where it should be done.

I've added more docs to spell this out explicitly.